pynicotine/gtkgui/ui/settings/colours.ui

```diff
@@ -22,7 +22,7 @@
     </child>
     <child>
       <object class="GtkCheckButton" id="DarkMode">
-        <property name="label" translatable="yes">Prefer dark mode</property>
+        <property name="label" translatable="yes">Enable Dark Mode</property>
```

Comment:
Since GTK is bundled with Windows installs of Nicotine+ (which I read is typical for Windows apps to each bundle their own GTK files to use), this option behaves as a light/dark mode switch. I will add a tooltip noting that the system's OS theme may take precedence.

I assume GTK being globally installed on Linux means an individual app cannot display in the opposite theme to the system's current one? However, that sounds wrong and like it should be possible. I think the ideal would be for this option to behave like a toggle on both OSes, and eventually introduce a "System theme takes precedence" option, once that can be read.

Reply:
While I would prefer this behavior, there doesn't seem to be a way to force a light theme. The only supported way is a "prefer-dark-theme" preference in GTK: https://github.com/Nicotine-Plus/nicotine-plus/blob/master/pynicotine/gtkgui/frame.py#L130

Reply:
I added a tooltip. It's mostly present to hint at the fact that to change the theme on some OSes, the user will have to switch the OS theme (i.e. if GTK is globally set to dark mode on Linux for example).